spec.html

```diff
@@ -20952,6 +21524,9 @@ <h1>Static Semantics: Early Errors</h1>
         <li>
           It is a Syntax Error if ContainsUndefinedContinueTarget of |StatementList| with arguments &laquo; &raquo; and &laquo; &raquo; is *true*.
         </li>
+        <li>
+          It is a Syntax Error if AllPrivateIdentifiersValid of |StatementList| with argument &laquo; &raquo; is *false* unless the source code is eval code that is being processed by a direct eval.
```

Comment:
```suggestion
          It is a Syntax Error if AllPrivateIdentifiersValid of |StatementList| with arguments &laquo; &raquo; is *false* unless the source code is eval code that is being processed by a direct eval.
```

Reply:
It's a single argument that happens to be a List.